arbnode/batch_poster.go

```diff
@@ -122,6 +127,9 @@ type BatchPoster struct {
 	batchReverted        atomic.Bool // indicates whether data poster batch was reverted
 	nextRevertCheckBlock int64       // the last parent block scanned for reverting batches
 	postedFirstBatch     bool        // indicates if batch poster has posted the first batch
+	fallbackToEthDA      bool        // use EthDA due to AltDA failure
+	ethDAFallbackCounter int         // counts batches posted in EthDA fallback mode
```

Comment:
nit: you could unify this with the boolean, where value==0 is same as fallbackToEthDA = false